tests/unit_tests/udfs/test_token_classification.py

```diff
@@ -114,92 +96,144 @@ def create_mock_metadata(udf_wrapper):
     SingleModelSingleBatchIncomplete,
     SingleModelSingleBatchComplete,
-    SingleModelMultipleBatchIncomplete,
-    SingleModelMultipleBatchComplete,
-    MultipleModelSingleBatchIncomplete,
-    MultipleModelSingleBatchComplete,
-    MultipleModelMultipleBatchIncomplete,
-    MultipleModelMultipleBatchComplete,
-    MultipleModelMultipleBatchMultipleModelsPerBatch,
+    # SingleModelMultipleBatchIncomplete,
+    # SingleModelMultipleBatchComplete,
+    # MultipleModelSingleBatchIncomplete,
+    # MultipleModelSingleBatchComplete,
+    # MultipleModelMultipleBatchIncomplete,
+    # MultipleModelMultipleBatchComplete,
+    # MultipleModelMultipleBatchMultipleModelsPerBatch,
     SingleBucketFSConnMultipleSubdirSingleModelNameSingleBatch,
-    SingleBucketFSConnMultipleSubdirSingleModelNameMultipleBatch,
-    MultipleBucketFSConnSingleSubdirSingleModelNameSingleBatch,
-    MultipleBucketFSConnSingleSubdirSingleModelNameMultipleBatch,
-    MultipleStrategySingleModelNameSingleBatch,
-    MultipleStrategySingleModelNameMultipleBatch,
-    ErrorNotCachedSingleModelMultipleBatch,
-    ErrorNotCachedMultipleModelMultipleBatch,
-    ErrorOnPredictionMultipleModelMultipleBatch,
-    ErrorOnPredictionSingleModelMultipleBatch
+    # SingleBucketFSConnMultipleSubdirSingleModelNameMultipleBatch,
+    # MultipleBucketFSConnSingleSubdirSingleModelNameSingleBatch,
+    # MultipleBucketFSConnSingleSubdirSingleModelNameMultipleBatch,
+    # MultipleStrategySingleModelNameSingleBatch,
+    # MultipleStrategySingleModelNameMultipleBatch,
+    # ErrorNotCachedSingleModelMultipleBatch,
+    # ErrorNotCachedMultipleModelMultipleBatch,
+    # ErrorOnPredictionMultipleModelMultipleBatch,
+    # ErrorOnPredictionSingleModelMultipleBatch
 ])
 @patch('exasol.python_extension_common.connections.bucketfs_location.create_bucketfs_location_from_conn_object')
 @patch('exasol_transformers_extension.utils.bucketfs_operations.get_local_bucketfs_path')
 def test_token_classification(mock_local_path, mock_create_loc, params):
-
     mock_create_loc.side_effect = fake_bucketfs_location_from_conn_object
     mock_local_path.side_effect = fake_local_bucketfs_path
 
-    executor = UDFMockExecutor()
-    meta = create_mock_metadata(params.udf_wrapper)
+    mock_meta = create_mock_metadata()
+    input = params.input_data
+    mock_ctx = StandaloneMockContext(inp=input, metadata=mock_meta)
+
 
-    exa = MockExaEnvironment(
-        metadata=meta,
+    mock_exa = MockExaEnvironment(
+        metadata=mock_meta,
         connections=params.bfs_connections)
 
-    result = executor.run([Group(params.input_data)], exa)
-    result_output = Output(result[0].rows)
+    mock_base_model_factory: Union[ModelFactoryProtocol, MagicMock] = create_autospec(ModelFactoryProtocol,
+                                                                                      _name="mock_base_model_factory")
+    number_of_intendet_used_models = params.expected_model_counter# todo is this always same?
```

Comment:
This depends on the test case, so having the number in the parameters is correct

Reply:
Only the name expected_model_counters is a bit odd. Maybe expected_model_calls or something like thst